### ISSUE TITLE
cannot open scripting console in pcbnew

### ISSUE THREAD

**Issue by adamwolf** [wayneandlayne/kicad-mac-builder#69]
n/t

**Comment by adamwolf** [comment]
🤦‍♂️ It may be because we *explicitly* install them in the wx directory, with `setup.py install --prefix=${wxwidgets_INSTALL_DIR}`.  Hmm, why did that happen?

**Comment by adamwolf** [comment]
On the latest version of the scripting branch, when I open the Python console in pcbnew, in the output I get:
```
Traceback (most recent call last):
  File "<string>", line 4, in makeWindow
  File "/Users/wolf/wnl/kicad/kicad-mac-builder/build/kicad-dest/kicad.app/Contents/SharedSupport/scripting/kicad_pyshell/__init__.py", line 221, in makePcbnewShellWindow
    pyshell = PcbnewPyShell(parent, id=-1, title=INTRO)
  File "/Users/wolf/wnl/kicad/kicad-mac-builder/build/python-dest/lib/python2.7/site-packages/wx-3.0-osx_cocoa/wx/py/editor.py", line 263, in __init__
    size, style, filename)
  File "/Users/wolf/wnl/kicad/kicad-mac-builder/build/python-dest/lib/python2.7/site-packages/wx-3.0-osx_cocoa/wx/py/editor.py", line 26, in __init__
    frame.Frame.__init__(self, parent, id, title, pos, size, style)
  File "/Users/wolf/wnl/kicad/kicad-mac-builder/build/python-dest/lib/python2.7/site-packages/wx-3.0-osx_cocoa/wx/py/frame.py", line 76, in __init__
    wx.Frame.__init__(self, parent, id, title, pos, size, style)
  File "/Users/wolf/wnl/kicad/kicad-mac-builder/build/python-dest/lib/python2.7/site-packages/wx-3.0-osx_cocoa/wx/_windows.py", line 580, in __init__
    _windows_.Frame_swiginit(self,_windows_.new_Frame(*args, **kwargs))
wx._core.PyNoAppError: The wx.App object must be created first!
```

Hmm.  I had this working in the prototype that inspired this whole refactor, so it isn't an inherent issue with the standalone python or anything.  Back at it, I guess!

**Comment by adamwolf** [comment]
Fixed in #72.
